.github/workflows/label.yml

```
@@ -0,0 +1,19 @@
+---
+
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: "Pull Request Labeler"
+on: [pull_request_target]
+
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v4
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

```suggestion
        with:
          sync-labels: true
```

Optionally, I think we could use the sync-labels option, and if it bothers us too much we (probably me, the first impacted), could remove it afterwards. 
https://github.com/actions/labeler#inputs